gui/support/views.py

```diff
@@ -193,7 +193,7 @@ def ticket(request):
 
         license, reason = utils.get_license()
         if license:
-            company = license.customer_name
+            company = license.customer_name.decode()
         else:
```

Comment:
I found another occurrence of this in a template: https://github.com/freenas/freenas/blob/freenas/11.0-stable/gui/templates/support/home.html#L58.

Should we take care of that too?